typescript/src/v0/types.ts

```diff
@@ -23,7 +23,7 @@ export type ChannelInfo = {
   topic: string;
   messageEncoding: string;
   schemaId: number;
-  metadata: [key: string, value: string][];
+  metadata: Record<string, string>;
```

Comment:
I think we probably want to use Map for these instead of Record. Especially since we are already using Map for some other fields like messageIndexOffsets.

Reply:
Yes you're right. I've changed this.